app/models/geo_entity.rb

```diff
@@ -15,7 +15,7 @@ class GeoEntity < ApplicationRecord
 
   has_many :country_citations, foreign_key: 'country_id'
 
-  scope :countries, -> { where.not(iso3: nil || 'GBL') }
+  scope :countries, -> { where.not(iso3: nil).where.not(iso3: 'GBL') }
```

Comment:
This is clearly a bug in the code as the developer must have intended to check for both nil or 'GBL' as writing nil OR 'GBL' makes no sense